src/main/java/bio/terra/service/iam/sam/SamIam.java

```diff
@@ -272,11 +272,16 @@ public void createProfileResource(AuthenticatedUserRequest userReq, String profi
     }
 
     private Void createProfileResourceInner(AuthenticatedUserRequest userReq, String profileId) throws ApiException {
+        // TODO: For now we continue to give stewards access to all profiles. That is consistent with
+        //  the current behavior. When we do the migration to the new permission model we should remove
```

Comment:
Is there a way to track this so it's easy to find when that future work is done? E.g., if there's a ticket for the migration to the new permission model, the ticket ID could be added here.